zcash_primitives/src/transaction/tests.rs

```diff
@@ -23,6 +23,18 @@ use super::{
 #[cfg(feature = "zfuture")]
 use super::components::tze;
 
+pub fn create_test_asset(asset_desc: &str) -> orchard::note::AssetBase {
+    use orchard::{
+        keys::{IssuanceAuthorizingKey, IssuanceKey, IssuanceValidatingKey},
+        note::AssetBase,
+    };
+
+    let sk_iss = IssuanceKey::from_bytes([0u8; 32]).unwrap();
+    let isk: IssuanceAuthorizingKey = (&sk_iss).into();
+
+    AssetBase::derive(&IssuanceValidatingKey::from(&isk), asset_desc)
+}
+
```

Comment:
Insert new work at the bottom of the file, not at the top. (the entire new function)

Reply:
Done.